subgraphs/_reference_/src/common/utils/logger.ts

```diff
@@ -0,0 +1,46 @@
+import { ethereum, log } from "@graphprotocol/graph-ts";
+
+/**
+ * Logger class is used to log messages to be picked up by our logging system.
+ *
+ * @param event The event that triggered the log message
+ * @param funcName The name of the function that triggered the log message
+ * @param msg The message to log (add `{}` to the message to add additional arguments)
+ * @param args Any additional arguments to add to the msg (same format as graph-ts log functions)
+ */
+export class Logger {
+  event: ethereum.Event;
+  fnName: string; // this will act as a rudimentary stack trace
```

Comment:
rename to funcName to match param doc above. or the other way around. I prefer funcName personally.